traffic_portal/app/src/common/modules/table/cacheGroupParameters/TableCacheGroupParamsUnassignedController.js

```diff
@@ -17,13 +17,15 @@
  * under the License.
  */
 
-var TableCacheGroupParamsUnassignedController = function(cg, parameters, $scope, $uibModalInstance) {
+var TableCacheGroupParamsUnassignedController = function(cg, allParams, assignedParams, $scope, $uibModalInstance) {
 
 	var selectedParams = [];
 
 	$scope.cg = cg;
 
-	$scope.unassignedParams = parameters;
+	$scope.unassignedParams = _.filter(allParams, function(param) {
```

Comment:
for my knowledge why?

Reply:
Because it saves me work :P

Everything we're using `_` for can be done natively in Javascript in every browser with above a 0.5% market share - except Internet Explorer and Opera Mini, both of which we don't support.

So I have a PR open to remove our dependency on it: #4410 .

[It's also every so slightly faster because it is built in](https://jsperf.com/array-prototype-filter-vs-filter).